src/grdimage.c

```diff
@@ -33,7 +33,7 @@
 #define THIS_MODULE_LIB			"core"
 #define THIS_MODULE_PURPOSE		"Project and plot grids or images"
 #define THIS_MODULE_KEYS		"<G{+,CC(,IG(,>X},>IA,<ID"
-#define THIS_MODULE_NEEDS		"Jg"
+#define THIS_MODULE_NEEDS		"g"
```

Comment:
I don't think you can remove `J` from `THIS_MODULE_NEEDS`. There are some explanations at https://github.com/GenericMappingTools/pygmt/issues/1493#issuecomment-921267870.

Reply:
Ok, thanks @seisman. I will restore J.